docs/index.html

```diff
@@ -4,7 +4,7 @@
 
 <div class="announcement">Sep 4, 2020 — <a href="/2020/09/04/leaflet-1.7.1.html">Leaflet 1.7.1</a> has been released!</div>
 
-<p>Leaflet is the leading open-source JavaScript library for mobile-friendly interactive maps.
+<p>Leaflet is the leading open-source (<a title="Leaflet's open-source license, BSD 2-Clause" href="https://github.com/Leaflet/Leaflet/blob/master/LICENSE">BSD 2-Clause license</a>) JavaScript library for mobile-friendly interactive maps.
```

Comment:
Please revert this change — we don't need license indication in the first sentence of the website.

Reply:
Removed as requested. 

If you don't like it as is, could you propose where the license in the docs to be referenced? Like at the end of Get Involved maybe like this "Leaflet uses the BSD-2 Clause license" as before, with a link to the GitHub license?